java/arcs/core/crdt/CrdtSet.kt

```diff
@@ -262,6 +262,37 @@ class CrdtSet<T : Referencable>(
             override fun toString(): String = "CrdtSet.Operation.Remove($clock, $actor, $removed)"
         }
 
+        /**
+         * Represents the removal of all items from a [CrdtSet]. If an empty [clock] is given,
+         * all items in the set are removed; otherwise, only those items dominated by the clock
+         * are removed. This allow actors with write-only access to a model to operate without
```

Comment:
Don't you still need to sync the clock for adding/removing?

Reply:
I don't think so, but good catch - I haven't addressed this.

Reply:
For future reference: adding and removing with a write-only handle will update the version map (clock) with the local actor's count without syncing the full map with any other CRDT node. Clear ops from a write-only will explicitly set the version map to be empty.